src/tree/PostgresAccountWizard/IPostgresWizardContext.ts

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Server } from 'azure-arm-postgresql/lib/models';
+import { IResourceGroupWizardContext } from 'vscode-azureextensionui';
+
+export interface IPostgresWizardContext extends IResourceGroupWizardContext {
+
+    serverName?: string;
```

Comment:
Rename this to `newServerName` to be consistent with our existing patterns and make clear it isn't related to an existing server

Reply:
What is the purpose of this? Given that the usage of this interface is for creation purposes, do we need to specify that it is a new server?

Reply:
Yes it is best to follow existing naming patterns even if the situation doesn't apply to this specific wizard (yet). There are many wizard steps (i.e. resource group) where the user can create a new one or select an existing one - which is why we follow this naming pattern